### ISSUE TITLE
Notes Sidebar should have landmark "complementary"

### ISSUE THREAD

**Issue by jeremy-w** [jeremy-w/orthic#25]

**Comment by jeremy-w** [comment]
This is now fixed - switched to a fork of the theme where I've fixed this.
